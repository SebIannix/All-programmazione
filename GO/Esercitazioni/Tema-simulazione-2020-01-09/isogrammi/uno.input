velocista
tavoli
supercalifragilistichespiralidoso
The Quick Brown Fox Jumps Over The Lazy Dog
conquista
break-time
dove si cena?
come si fa?
isogram
six-pack
casa
tavolo
cruscotto
diavolo
