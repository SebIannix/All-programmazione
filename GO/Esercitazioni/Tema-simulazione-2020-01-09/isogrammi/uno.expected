velocista: SI
tavoli: SI
supercalifragilistichespiralidoso: NO
The Quick Brown Fox Jumps Over The Lazy Dog: NO
conquista: SI
break-time: NO
dove si cena?: NO
come si fa?: SI
isogram: SI
six-pack: SI
casa: NO
tavolo: NO
cruscotto: NO
diavolo: NO
